tests/test_models.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+
+from booking.models import Event, TicketOrder
+from datetime import datetime, timedelta
+
+
+def test_event_id_is_not_mutable():
+    event = Event('Rock Live', 50, 229.99,
+                  datetime.utcnow() + timedelta(days=1),
+                  datetime.utcnow())
+    assert event.id is not None
+    with pytest.raises(AttributeError):
+        event.id = 5
+
+
+def test_order_id_is_not_mutable():
+    order = TicketOrder(None, 5, 'Client Name', 'Somewhere, US',
```

Comment:
There also should be some kind of validation that event name can't be empty, then we don't know what kind of tickets do we book.